From marcin.slusarz@gmail.com  Fri Jul 10 16:05:31 2009
From: Marcin Slusarz <marcin.slusarz@gmail.com>
Date: Wed, 08 Jul 2009 22:07:07 +0200
Subject: Staging: rt3070: add support for Linksys WUSB54GC-EU v3
To: Greg Kroah-Hartman <gregkh@suse.de>
Message-ID: <4A54FC6B.7000003@gmail.com>


According to FCC (*1) and *2 there's really RT2070L inside, but according
to *2 RT3070 and RT2070 are compatible.

*1 https://fjallfoss.fcc.gov/oetcf/eas/reports/ViewExhibitReport.cfm?mode=Exhibits&RequestTimeout=500&calledFromFrame=N&application_id=327783&fcc_id=%27Q87-WUSB54GCV3%27
(look for "Internal Photos")
*2 http://rt2x00.serialmonkey.com/phpBB/viewtopic.php?f=5&t=5245

Signed-off-by: Marcin Slusarz <marcin.slusarz@gmail.com>
Acked-by: Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rt2870/rt2870.h |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/rt2870/rt2870.h
+++ b/drivers/staging/rt2870/rt2870.h
@@ -219,6 +219,7 @@
 	{USB_DEVICE(0x0789,0x0163)}, /* Logitec 2870 */		\
 	{USB_DEVICE(0x0789,0x0164)}, /* Logitec 2870 */		\
 	{USB_DEVICE(0x1EDA,0x2310)}, /* AirTies 3070 */		\
+	{USB_DEVICE(0x1737,0x0077)}, /* Linksys WUSB54GC-EU v3 */	\
 	{ }/* Terminating entry */                      \
 }
 #endif
